### PR TITLE
fix format of man page

### PR DESCRIPTION
**Pull request by WeiZhang555**
This commit fixes some format problems of man page, most about the
`docker network` command, also add some contents and fix minor typo.

Signed-off-by: Zhang Wei zhangwei555@huawei.com

There are still some other format problems which can't be solved by editing the `md` files, I'll open a new issue to illustrate it.
@cpuguy83

### CODE REVIEW

**Comment by WeiZhang555** [comment]
The other related format problem is given in #17469

**Comment by WeiZhang555** [comment]
ping @cpuguy83

**Comment by thaJeztah** [comment]
Thanks @WeiZhang555, I'll review this later!

**Comment by WeiZhang555** [comment]
Just append one more commit based on previous one, for solving the format problem bring in issue  #17469 (item 3: underline thing).
Now I force the OPTION value to use underlined format, so we can unify all the man pages format.
It doesn't change original contents much, most is about format problems.

**Comment by LK4D4** [comment]
ping @thaJeztah @moxiegirl

File: man/docker-cp.1.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ docker-cp - Copy files/folders between a container and the local filesystem.
 **docker cp**
 [**--help**]
 CONTAINER:PATH LOCALPATH|-
+
+**docker cp**
+[**--help**]
```
Comment (thaJeztah) [inline man/docker-cp.1.md]: Just wondering if this is possibly confusing because we're showing the same command twice here

**Comment by thaJeztah** [comment]
thanks for the ping, @LK4D4. Sorry for the delay, @WeiZhang555 I thought I reviewed this one already :blush: 

one question/thought, but LGTM otherwise. Thanks for doing this!

Can you squash your commits?

ping @moxiegirl for review

**Comment by moxiegirl** [comment]
@WeiZhang555 very nice. LGTM

File: man/docker-cp.1.md (hunk 1 above)
Comment (WeiZhang555) [inline man/docker-cp.1.md]: Not quite the same, final look after change: 
```
       docker cp [--help] CONTAINER:PATH LOCALPATH|-

       docker cp [--help] LOCALPATH|- CONTAINER:PATH
```
And original is as following:
```
 docker cp [--help] CONTAINER:PATH LOCALPATH|- LOCALPATH|- CONTAINER:PATH
```

It's hard to understand for me, I think the change makes it clearer, WDYT? :smile:

**Comment by WeiZhang555** [comment]
The commits are squashed, thank you for help reviewing @thaJeztah @moxiegirl 
PTAL :smile:

File: man/docker-cp.1.md (hunk 1 above)
Comment (thaJeztah) [inline man/docker-cp.1.md]: Yes, I think it makes sense, thanks!

**Comment by thaJeztah** [comment]
Awesome, great job @WeiZhang555 :+1: 

LGTM

**Pull request merged**